Trell/CliCommands/RunCommand.cs

```diff
@@ -29,6 +37,28 @@ public override ValidationResult Validate() {
                 || !File.Exists(Path.GetFullPath(this.DataFile))) {
                 return ValidationResult.Error("Uploading requires a valid path for an existing file be passed as an argument");
             }
+        } else if (this.HandlerFn == "request") {
+            if (string.IsNullOrWhiteSpace(this.DataFile)
+                || !File.Exists(Path.GetFullPath(this.DataFile))) {
+                return ValidationResult.Error("A valid path to an existing file is required for making requests");
+            }
+            if (this.Url is not null && !Uri.IsWellFormedUriString(this.Url, UriKind.Absolute)) {
+                return ValidationResult.Error("An ill-formed URL was given as an argument");
+            }
+            if (this.Headers is not null && this.Headers.Length > 0) {
+                this.ValidatedHeaders = [];
+                for (int i = 0; i < this.Headers.Length; i++) {
+                    var str = this.Headers[i];
+                    if (string.IsNullOrEmpty(str)) {
+                        return ValidationResult.Error("Headers must be formatted like this: \"Header-Name: Header-Value\"");
```

Comment:
Looks like `curl` ignores invalid headers so `curl -H 1` doesn't pass 1 in the request, but `curl -H 1:1` does pass that as a header. @isaksky, complain or ignore?

Reply:
@oconnor0 I think complaining is good